conda_pack/tests/test_core.py

```diff
@@ -407,29 +459,36 @@ def test_dest_prefix(tmpdir, py36_env):
 
     # shebangs are rewritten using env
     with tarfile.open(out_path) as fil:
-        text_from_conda = fil.extractfile('bin/conda-pack-test-lib1').read()
-        text_from_pip = fil.extractfile('bin/pytest').read()
-        binary_from_conda = fil.extractfile('bin/clear').read()
+        lib1_script = '/'.join([BIN_DIR, 'conda-pack-test-lib1'])
+        pytest_binary = 'scripts/pytest.exe' if on_win else 'bin/pytest'
+        text_from_conda = fil.extractfile(lib1_script).read()
+        text_from_pip = fil.extractfile(pytest_binary).read()
 
     assert dest_bytes not in text_from_conda
     assert dest_bytes not in text_from_pip
     assert b'env python' in text_from_conda
-    assert b'env python' in text_from_pip
 
-    # Other files are rewritten to use specified prefix This is only checked if
-    # the original file did include the prefix, which is true at least on osx.
-    orig_path = os.path.join(py36_env.prefix, 'bin/clear')
-    with open(orig_path, 'rb') as fil:
-        orig_bytes = fil.read()
+    if not on_win:
+        # pip entrypoint on Windows is complicated...
+        assert b'env python' in text_from_pip
+
+        with tarfile.open(out_path) as fil:
+            binary_from_conda = fil.extractfile('bin/clear').read()
+
+        # Other files are rewritten to use specified prefix This is only checked if
+        # the original file did include the prefix, which is true at least on osx.
+        orig_path = os.path.join(py36_env.prefix, 'bin/clear')
+        with open(orig_path, 'rb') as fil:
+            orig_bytes = fil.read()
```

Comment:
If binary files with prefixes occur on windows, it would be good to test this on windows as well, rather than skipping it. `bin/clear` was chosen on unix/osx, as it always exists, and needs binary replacement. If this is needed perhaps there's a windows equivalent.

Reply:
We're testing binary replacement with pytest.exe